{"uid":"6be5206f2f66f199","name":"test_TC227_Verificar_error_al_crear_opcion_con_position_mayor_a_999999999","fullName":"tests.catalog.options.test_HU11_ES_POST_ProductOption#test_TC227_Verificar_error_al_crear_opcion_con_position_mayor_a_999999999","historyId":"09c4f63e9a9984c5936c41cb895fd765","time":{"start":1755467427530,"stop":1755467427690,"duration":160},"status":"skipped","statusMessage":"XFAIL Known issue BUG: Error 500 (Internal Server Error) inapropiado al validar campo numérico\n\nAssertionError","statusTrace":"setup_add_options = ({'Authorization': 'Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJpYXQiOjE3NTU0NjczNzAsImV4cCI6MTc1NTQ3MDk3MCwicm9sZX...hoV6i5jANuNmgvkWkcENYm6BfEqO_DBZZoQ7onruQ3z4X1x8eJp35FmlOffvMcgZywWXhKy0Foc10MT-l7iMeyKNSOFiM9xlQJClw0Q7U4Bf6uGI'}, [])\n\n    @pytest.mark.functional\n    @pytest.mark.boundary\n    @pytest.mark.xfail(reason=\"Known issue BUG: Error 500 (Internal Server Error) inapropiado al validar campo numérico\", run=True)\n    def test_TC227_Verificar_error_al_crear_opcion_con_position_mayor_a_999999999(setup_add_options):\n        headers, _ = setup_add_options\n        payload = generate_options_source_data()\n        payload[\"position\"] = fake.random_int(min=1000000000, max=10000000000)\n        response = SyliusRequest.post(EndpointOptions.options(), headers, payload)\n        log_request_response(EndpointOptions.options(), response, headers, payload)\n>       AssertionStatusCode.assert_status_code_422(response)\n\ntests/catalog/options/test_HU11_ES_POST_ProductOption.py:174: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [201]>\n\n    @staticmethod\n    def assert_status_code_422(response):\n>       assert response.status_code == 422\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^\nE       AssertionError\n\nsrc/assertions/status_code_assertions.py:59: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1755467370080,"stop":1755467370081,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"auth_headers","time":{"start":1755467370081,"stop":1755467370590,"duration":509},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"setup_add_options","time":{"start":1755467427530,"stop":1755467427530,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"testStage":{"status":"skipped","statusMessage":"XFAIL Known issue BUG: Error 500 (Internal Server Error) inapropiado al validar campo numérico\n\nAssertionError","statusTrace":"setup_add_options = ({'Authorization': 'Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJpYXQiOjE3NTU0NjczNzAsImV4cCI6MTc1NTQ3MDk3MCwicm9sZX...hoV6i5jANuNmgvkWkcENYm6BfEqO_DBZZoQ7onruQ3z4X1x8eJp35FmlOffvMcgZywWXhKy0Foc10MT-l7iMeyKNSOFiM9xlQJClw0Q7U4Bf6uGI'}, [])\n\n    @pytest.mark.functional\n    @pytest.mark.boundary\n    @pytest.mark.xfail(reason=\"Known issue BUG: Error 500 (Internal Server Error) inapropiado al validar campo numérico\", run=True)\n    def test_TC227_Verificar_error_al_crear_opcion_con_position_mayor_a_999999999(setup_add_options):\n        headers, _ = setup_add_options\n        payload = generate_options_source_data()\n        payload[\"position\"] = fake.random_int(min=1000000000, max=10000000000)\n        response = SyliusRequest.post(EndpointOptions.options(), headers, payload)\n        log_request_response(EndpointOptions.options(), response, headers, payload)\n>       AssertionStatusCode.assert_status_code_422(response)\n\ntests/catalog/options/test_HU11_ES_POST_ProductOption.py:174: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [201]>\n\n    @staticmethod\n    def assert_status_code_422(response):\n>       assert response.status_code == 422\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^\nE       AssertionError\n\nsrc/assertions/status_code_assertions.py:59: AssertionError","steps":[],"attachments":[{"uid":"decf00866167447","name":"log","source":"decf00866167447.txt","type":"text/plain","size":2192}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":1,"hasContent":true},"afterStages":[{"name":"setup_add_options::0","time":{"start":1755467427697,"stop":1755467427697,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"labels":[{"name":"tag","value":"boundary"},{"name":"tag","value":"functional"},{"name":"parentSuite","value":"tests.catalog.options"},{"name":"suite","value":"test_HU11_ES_POST_ProductOption"},{"name":"host","value":"pkrvmubgrv54qmi"},{"name":"thread","value":"2375-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.catalog.options.test_HU11_ES_POST_ProductOption"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"tags":["boundary","functional"]},"source":"6be5206f2f66f199.json","parameterValues":[]}